### DISCUSSION TITLE
Upstream DNS servers

### DISCUSSION THREAD

**Discussion by gyCfjSnO** [AdguardTeam/AdGuardHome#3153] [Q&A]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * v0.106.2
* **How did you install AdGuard Home:**
  * Ubuntu
* **How did you setup DNS configuration:**
  * Router
* **If it's a router or IoT, please write device model:**
  * Technicolor DJA0231

### Expected Behavior
Allow me to use TLS/HTTPS DNS Upstream servers

### Actual Behavior
I am getting the error "Server "tls://dns.google": could not be used, please check that you've written it correctly" when testing the TLS/HTTPS DNS Upstream servers, Using a plain regular DNS (UDP) works though. This occurs for ALL tls & HTTPS servers I am trying.

**Comment by RonKhondji** [comment]
tls://8.8.8.8 does seem to work just fine.
